dev-packages/node-integration-tests/suites/tracing/apollo-graphql/scenario-mutation.js

```diff
@@ -39,5 +39,4 @@ async function run() {
   );
 }
 
-// eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
why is this rule no longer applied? (no objections here since it's a test but we still want this rule in src files)